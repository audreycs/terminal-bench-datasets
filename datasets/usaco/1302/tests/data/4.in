100

5 292 250
745647896 502754032 303840855969
398461622 241872325 122957508729
37870821 833783293 88458153966
671875747 718231776 192620073816
996462649 36814364 150267137027

7 849 760
501731268 153881024 522202065380
745871782 65947687 284591269896
860311832 717900135 655422699934
539020288 428943669 404770038725
36300387 310153139 185285836872
225666257 221071796 266454214458
173879290 722501674 591838140130

9 321 839
885365224 911709360 881567603690
987496437 398509243 467198724013
697589685 511940871 615069935508
537455599 251830669 322845310491
326100745 148861662 206974094270
22573633 518541084 430457324676
900351636 863428369 885488911693
494056345 701134152 666405207479
532296848 61093026 126376486539

7 417 377
42306925 782103746 182441833830
35614099 973823992 377270274585
308482292 800351068 208972809913
893395368 611607573 448880860512
753051759 195125290 291819560783
127193615 481243699 180125440946
940341515 779665409 409738843342

6 402 404
568826089 893018533 560912945120
960185749 618669492 556469253507
329362172 649271547 390226314327
44614845 409049940 139943235197
93657406 698434127 303159919922
995088244 325314468 471714184897

10 79 307
801331375 259116381 129648211157
169942453 652919752 194811692113
353917998 311443982 115121805674
295428362 269152014 99625352720
622489094 975735745 329651502782
823263980 514207606 207572348610
530186414 344607646 131364685212
161386182 655787960 205926171461
175631501 432499118 138928092839
365693157 920102039 304366962417

5 637 533
73092435 369041116 232289400754
31053183 196642883 108798244311
606248143 981664634 749997075146
283664674 431334971 335469814015
997116893 931683175 957523135034

7 273 309
711487032 5410230 169312153385
926825142 194076604 217462582821
165592136 384942247 134329146298
685519959 725383882 307450305112
14439589 916508282 276809095923
285638139 493696030 220264720264
343412298 162802672 121098565510

9 8 434
881761501 121136783 58626520686
345584935 396726504 146547719326
403782645 931913342 332840793530
901605285 208199686 80910564954
500546793 292052169 124348210498
303217812 749053460 316993890242
878778776 200966496 77069970830
759219692 3561683 6604663529
380844644 156743158 68105412453

8 97 679
240376073 727248105 382846272660
825172293 157703465 157938000735
792247591 382697045 293589752742
373760051 995909696 629076153488
910730743 312595132 275350531316
763677556 464081849 271061715829
449795292 6137331 42225743302
339696786 581336660 386867490310

5 53 119
367791664 369061277 49378292738
286581953 197515825 29204462549
12297598 9053466 1262297519
398256978 637478406 88927304834
163370908 201476327 26515008237

7 37 682
641940137 930826376 451813315321
67576026 35951114 20482288327
660688033 896710314 279825511140
984955450 362485558 174776357019
379594099 765104352 98337026817
64963825 282307219 62035784398
333722719 799449053 495967990732

6 955 768
971710336 1313959 864191970936
653667472 934446431 709570228442
436098230 568710113 346000132636
896443394 330698127 740127624269
159775720 933544429 585017580004
213575205 798859158 556955984868

6 485 388
989237712 371655658 543746484859
897036334 310455588 408846713597
772958804 317957741 327314973585
200236953 317490132 139699828760
243062268 505875186 296201294753
339521997 108392006 137945208797

6 810 718
841309613 280739629 830640056689
115865725 853347968 600714512220
47360511 18933178 35825489413
334851131 546538871 285386366482
276408054 613606466 605597286597
349471649 573992649 530639786370

10 489 818
574589232 802030699 891159991651
622652382 876496181 812142456941
433144356 824284190 849788733864
796995876 969249221 1149383580880
960643763 671476944 856003646564
958772118 249638905 394364109060
103912898 550664678 483755445033
75787863 171366249 158581679982
459682068 860888916 873364645920
822389823 334289958 520192312662

5 388 786
286509425 478691576 417674703007
330152399 484351689 384625889267
417864066 254038390 170578218217
964155197 17047185 135442638260
828666734 298814406 268078826536

8 713 918
161609373 508924940 563250081239
718394018 402758872 812542273304
977158120 856340321 1419950130330
901245203 778643232 1339507248460
21249798 344435079 323594068396
927944848 57353416 679045790126
954161450 737516689 1337096041341
429683243 671947124 889889163444

7 494 735
799382391 475973876 462623672841
226983173 540262632 230459168493
633784173 371556589 353696662040
333424196 459443562 173062624800
800671265 641310004 577238103711
853079800 242412235 415716566659
33769238 260705535 65113366193

7 535 779
334929904 755623797 684292637317
45290046 22848603 40427930767
767844202 897886290 738050152946
650890001 275396202 530239165194
185748563 617898425 296940627493
595095790 618549665 745486554602
917111534 353005557 417591344091

5 28 778
720486204 144515785 121864350160
743693549 126114610 112878598920
91272249 316031871 239680940728
532878710 932566420 712579576528
677798898 683017790 500188443781

9 436 465
222689429 680854225 363363377145
208343011 245279733 172860290819
960570508 477707299 551166558616
499714638 708929922 420816216150
94739871 805041324 345444002424
651538834 321311399 404661403314
763247688 761138574 588141712447
680622634 646624474 506249109785
970786867 788727174 698093660723

8 754 644
352099939 593475977 627361655202
502082168 536580222 708804182580
990367395 480425113 664050602067
709324638 460073367 355970029509
584669080 20782581 264208194945
662189445 970289162 391067704834
111296100 221166612 57750260133
490167457 981867268 870556643482

6 288 972
871458647 863413025 1075807542073
159586610 691127022 703363891509
150028093 322842871 265081973702
285705791 717456861 485406189136
998111447 716884593 873604181713
966139682 714253578 893415287151

9 232 482
285878106 349965808 90649710411
341516169 428254827 204041847191
403461016 693595069 198355092251
133364143 249867903 108754645375
457034943 351406969 158849195367
963403427 600948949 108667192726
72003687 895190140 275686335829
866405592 364761697 352600518977
908359944 350647910 145006434439

5 871 375
843039923 871903851 751131073547
983901118 908702306 1089469064918
30005558 958364255 85515673585
776336366 39847729 666646913348
540097452 142257965 479265460321

5 994 317
124137293 546913378 241434806355
372389001 305260853 422141527418
219521845 551372430 392662837794
438354909 17172965 429403354843
342808607 617958834 513326947054

7 586 751
427820486 504797718 507896416479
875424790 16000711 105391414339
898522585 83817882 360708718046
13994691 730676521 543762166759
55143816 948036406 723309963085
713731453 821841124 984145838153
224123013 127385446 155562442641

9 44 755
546712969 252260197 101349083181
85549098 340038616 235250832675
990757088 746866035 352633325608
978334207 763108469 400671881425
549392899 944887899 573609497842
478384904 872938 14866549435
67691765 925409012 291497260104
844464031 362525829 293563175668
256640677 982482100 375329407438

8 754 813
820240906 197260172 603752699910
308078574 872015467 786375904750
412305033 559433146 478782757889
783215019 622483156 598039240439
868525046 943043480 725897722303
364049129 63337717 294138414886
272636704 555518674 574029404830
623111506 247305558 432819008614

8 62 464
561383293 123323850 41277902931
515442546 596002045 247543962736
727947544 102500838 55034413446
17726466 927688961 250733439969
509849414 588290702 271155352183
426114694 817086565 283763564142
604657270 165321707 76096732040
413017448 560594090 184180402745

5 558 477
799772680 370751217 497484388265
806445211 331472322 544295707889
278877070 250037646 198940252729
848570248 455401567 480145865469
63333956 548113225 221372710125

5 459 796
855847174 735460351 618460102757
242754406 1096685 44821280122
228838204 992666790 629789978240
748820669 189349935 160550408538
884570251 538409918 722983876611

10 712 504
889404294 460060216 758010409955
834967909 437681969 754409727361
660299888 695923464 672341733918
525892225 709563477 729965002321
689044277 552497205 740999633344
108003379 709797157 309961474478
164056219 714573594 446590668744
504839828 695492210 606996885109
737862663 793862452 774273314641
719385739 670545509 718536321201

9 586 414
538687992 966848376 492062159069
216554524 757635685 162432478324
43836597 93381229 21568846801
556689534 576081882 551135031662
222461568 863623359 224131144302
700110719 315317637 291891323407
248536087 467876680 108218235283
32729732 605983096 107842676479
891270973 519525006 247349969362

6 894 959
44770987 527117607 486310072472
381473363 573503749 655551323599
489440977 325340061 611940582356
909856569 527870497 890018403801
84360406 636003652 648632725814
991270161 353877497 1155258111725

5 435 306
258919996 960383637 245364559805
625951504 20948384 270583319571
193134928 575255834 201034842846
595320690 185974813 302865591578
245788316 629154778 230229188926

8 88 901
286096443 742260567 623433382306
273245836 309554620 228897498817
926908863 923916416 852002824621
669796757 998612638 906436022033
51752445 172422427 92510192251
300852105 546245908 517593456775
478671143 354827353 329761102248
96363533 332899580 168038625038

8 505 378
988157969 971764914 637553444332
955903275 242618966 480353392309
49687242 140831967 53299023682
148470316 216329179 140066219419
255680355 987615679 496346638130
463939902 53369694 187774691822
528636499 956103648 607935489712
696689911 817235437 475431985734

7 34 137
46811667 465485364 51357140810
24136240 133184586 5112170016
274762735 567804791 69902287137
340805690 231049899 20894269899
840031231 426935695 67024262092
560069064 714897295 53816538035
990856898 476368228 88184237436

7 478 166
4314826 836539188 123851900762
483030733 46098130 176634004188
450586624 353722327 199616408229
644936643 308641965 256880990437
12788135 40159041 11169650497
295636474 709726891 224978330239
398479172 851507322 302420997353

5 238 881
436984775 133000830 218376284433
518531880 529699518 561982558206
923364127 345674788 520768718240
372190014 423084517 458329466191
193353154 703569827 659356965202

10 896 296
173159237 386122259 122778787415
990970159 17262627 795584215818
476839029 470364805 201711456308
939023475 97309108 453609739795
684090629 721915058 784330924748
250012906 626597252 176968427940
152246845 21243174 124340276107
949724592 730553242 337270251260
311291585 186593413 216541927804
788950598 482905643 664220285132

7 132 563
82531135 825563565 423204704073
339984515 804290350 487318374131
265143690 936826997 522548664944
61458551 618182409 332528421668
522436093 206986559 169701375302
480607759 239131104 171987122022
189596357 940690473 531442086659

10 312 383
657232197 50218588 215111520649
863341778 19250771 224819770370
857016212 435899441 420880998859
547946178 499805788 345956066574
923890568 952084347 518808045747
759390186 930548664 328332379107
128421351 599509625 106643548118
77516067 850050717 202142239615
270274691 459637187 133412000715
765125902 7661026 174354127933

5 554 773
384270568 707346904 717074860693
486874118 837566423 547218626141
286725667 232462711 225492026563
121190296 355491053 250101065688
505641796 606943687 684011903980

10 783 351
941098684 155002138 686132996074
734307249 919404828 590399916809
72527021 999618055 392465754899
846518712 489485442 764015380991
567694998 276179152 346751890059
407875056 125572987 289794812225
317466849 956279441 488621270599
690826213 114782498 401505860593
601164398 54322657 224744259535
248961584 661969220 363902532944

5 244 941
532632896 794590917 709852407095
458208363 678740204 745244216298
120758844 955166576 612402469066
78315436 991357941 705750926360
614677384 636524569 686630986905

7 772 849
261811099 568974462 542107879763
12638642 438920583 341763944590
244836293 839823559 591262665952
570140104 153465196 497330112107
910033666 572836128 1129793282727
557266100 556143426 779483124037
694210953 210445552 605474319391

8 935 49
286257631 145716530 243513084846
475374279 130936309 54049324480
892830392 833333776 582670211655
103718422 574280392 27015858146
724942854 29533378 349573164898
556234359 731932271 444821608820
573557461 830965329 365675533398
988513110 300069152 184320004482

5 966 406
835379975 995111276 829352536292
596376786 483492743 570535736384
989960603 491114072 977774133283
727545107 934435478 650904408173
946833947 771841387 773860155465

6 914 809
516758866 387552615 727717101353
520006382 246150909 615107839261
64042542 47141221 87304789022
382915897 492950726 662933350703
492459142 472135569 668726055511
691624454 838856420 1294419623149

9 359 757
190463504 105896275 53274732128
898140392 690950125 487788035720
241970583 766837542 219111295490
821767434 673456824 439632017295
461707723 672639962 411966063487
486143489 649594100 362638220460
960299546 775971345 463276106991
41331417 237095273 70980450068
119101878 655562649 468984834015

7 144 654
519393526 126997320 156238028640
501799735 312268559 272949226467
926426596 824876813 634461035564
316016746 633154672 427126915633
162789199 630780344 424841248091
967727341 687992990 586914668124
84602820 648183127 410484637278

8 443 807
28346589 621711597 446473624426
418798185 455352374 421625822851
742176626 7093340 191931182902
808883587 981711750 1105850626388
439328510 329477514 308817992556
961683486 852636498 786567983844
514707471 297711292 307070725282
363214526 822389012 684660628856

7 308 823
535181409 121257837 134944282371
71309927 798717480 592431763762
463303985 257282906 174975952581
634681191 789099790 628005022704
599029677 527848631 302183308735
118214201 526347570 336326008508
196336410 527959036 478305873064

9 898 643
508668425 653698445 425074006234
223911466 649043552 331625610648
518083279 162784114 299917292632
5351772 314970865 73906729687
950686601 721908375 1257760219380
569452261 109540482 421586665617
379518138 991472627 551081802839
35695773 33127532 29377626080
956106035 413933835 799902947170

7 747 990
410838874 453966894 480007602695
952436014 495104799 774375046330
642835173 591150359 728586114756
993987074 738769322 752672063703
327935016 318943846 478828567889
68619926 655589974 293647637999
681949659 788489259 1194718034243

8 979 292
88925002 559153934 219079391824
385354757 548256749 501480980215
395028067 107451779 280192817000
936508289 89836506 771527392215
510601699 304667466 553448232619
267400963 223684119 281717517449
919218021 900177814 1008664057140
279787853 684262840 400989637403

9 291 846
919198249 717019337 863827749716
299376533 48031472 77256770467
805331356 882344609 667840860172
106602411 435664788 112361645639
434937819 906301629 350637568404
783730427 769831222 539177054027
932721477 928169195 387224550850
790048597 399457622 563192403054
790225738 841922248 693695866037

6 760 288
79201102 758976768 146765063873
542771177 69873023 386843849373
899343770 149908882 424460469402
343072627 267390128 200083385779
978624772 963238839 536397968139
236403744 242156518 165342033000

8 625 933
266920584 505425316 612970724576
902731629 998965535 1372421363438
752925765 44791671 320549584322
795600835 814946229 1176409466454
769262728 864068355 1201993606637
381471882 420668909 562991757601
216746598 822582567 775847289096
593431943 80255290 360348038633

7 375 582
445680704 45817607 119451308533
464894300 264903555 318901912356
789594900 499358803 404457589149
439119221 525847392 206628445165
710227452 620141862 458979266478
142888777 322339087 129906772607
306813759 271941504 178435089041

9 539 145
334923100 961421094 115160994891
560218271 661590633 239629331911
81731116 915099675 128058054173
825099413 585311033 432785817949
713011091 799229237 312038486911
463738983 387590740 75548301157
854068164 475540069 318048770804
491400736 862506627 325311303331
419691784 24496435 134329668625

8 922 94
27243156 78578283 22773935281
56683761 681290705 104967041741
251521765 814733437 234098321961
692319309 151883710 438310101330
829788021 739783410 446343639196
596390665 412442401 208533091737
192398017 198796439 147112188299
548425327 455447765 131259492483

8 456 564
347404238 993127971 348055483593
153792097 871350803 169407317531
901970972 537864212 529990138515
101920000 833786637 77922789988
556591158 731959285 171427667488
927852087 268147664 456166731133
923581280 235587233 411078773411
754623131 973297996 313759235449

6 511 834
734564346 85425858 438695837484
578126376 751172964 794887030520
3867130 68985493 39254026355
744436353 406476041 437416474637
547385851 922298247 1031438669096
451740828 707764924 317683416963

10 230 281
737281114 518196528 107096454155
718997107 939458093 422934162709
961008109 24242881 166283207352
120412461 684746662 217894626997
526775309 516044952 201455795445
71276369 597093591 131259215812
488719640 573450857 188262448657
682214862 180697014 78127157756
589799665 971218799 325649707863
522848716 520313346 243194979380

6 979 335
765733654 442850660 643387569127
708227829 127004398 547535052115
420994209 905155438 419871301684
210770713 75971710 122300381900
603057760 608237829 656525435160
440423689 429005469 430548240125

10 979 197
982957846 555124719 652744138340
537289935 592653898 202182275156
694244270 412655762 597153787277
218405251 972404648 311082422349
382803990 202568087 47243225339
302889747 69797358 178745088760
473869347 219702174 213950034658
375138489 945812531 288081157779
187953513 659086272 178301007241
296347115 302967832 68581757509

6 700 346
416440270 590786609 452587285827
968097487 129914262 713619342060
294915949 338841742 217200160062
714758218 892848109 806026557867
426432464 898175003 354971298704
299115117 775288245 376821360737

5 862 568
199548642 601725280 217457652030
525750284 410338497 230572959798
101203693 53182398 111045828712
953119201 347057254 966759670099
225210887 406949428 138363402875

9 798 620
717503133 719485927 760145757990
988319973 771091612 557080312091
359747762 940485908 624571680265
204275784 926745148 599442180468
81286533 224699140 202054573429
734372984 226459025 720155981872
647157278 271942123 530127525765
171920701 785109584 586872858950
372260636 872844236 789435396999

10 409 155
901599605 62025558 317355826806
678534804 329542625 143889226876
506814413 675206028 230161878744
792292741 417387912 264890978054
391347058 924715514 250560547061
567427483 379152990 110164216556
5554369 350913530 39408271883
730931393 219951116 115177500161
159087153 482411784 73778164193
452945870 987852616 146218744363

5 81 724
316118003 988995108 685828340647
570017157 935071472 372683718166
722551467 150658739 138280428960
792928578 530337306 354946224807
796434043 475527253 88869303718

6 715 133
446303276 812076793 269120718744
26138240 25260297 15175171999
467410329 486273975 267741111477
399977234 322436526 297987480761
304627800 724522844 94587823855
565095624 919231675 276791973076

8 8 209
898660148 75478938 19116964823
251406652 589640021 109587065378
516112183 212302003 38870223470
738122656 733912340 143507821889
963304862 89276193 18928557402
440375973 28131752 6929080878
682672210 868759924 153785202385
112386052 14025923 2984584738

7 315 841
614962236 416744355 530395538641
768379953 873815533 970065207107
227764549 1922825 67114789479
811887968 437595493 305582433424
125493341 681417330 541122964153
359165628 640475359 549556544807
633706524 208539324 217295847013

10 87 146
413748456 387118395 55399762267
256941357 81545724 25496765241
326854292 381554508 80502498980
382141232 515720455 83968409879
991917039 272508545 98501143382
310277607 853635328 148581870118
831725025 555828502 101180351592
98226167 19567580 4624288685
702273226 498145723 83072130877
78659146 38674159 8521555782

10 84 527
206965435 710079694 342592106856
99542315 381750972 117669934511
508319877 70371680 60540026525
971679559 336436068 191075231708
686596705 665249617 291004942378
446665795 807917767 307952015142
260126705 347918136 134191178482
231652490 604237864 215288735771
376140434 646590616 349417196367
761652271 63728559 94169751165

7 7 470
749883931 669857 2296197788
253669842 208251495 38328992967
750622494 637608042 289604136433
380926952 192628836 89251255641
368817509 928619118 189462401460
398110837 224970442 27549887148
384603165 382080356 118138748820

9 618 928
96607227 371254682 137973054483
211165456 18253116 59482656556
201291512 679201604 249470480117
265432507 745133226 305382712771
302596526 58528093 199396602737
507268051 480713679 473922110575
479588744 523294592 692835285251
202963119 74716026 76853993543
292163369 172693764 209551894599

9 213 581
984746541 251805343 308423805662
184594976 348630442 237114151506
364796979 354494000 276284480188
298131355 261729967 202167935626
716214434 419272211 340118017029
928678674 208288640 239972685793
624502431 787685254 583524239704
66525428 64201740 49290634427
565484212 903316087 623005827145

10 418 339
226540073 323856651 201078160890
915474807 294685172 367436222027
772973680 872782193 477955145764
897049180 757193120 341468153894
886211673 508735171 445024787958
448060913 997208579 414614644631
21050208 882251910 305967095548
855891844 751115874 338947423448
142937989 88456262 69015285580
558673000 513439818 341277408353

10 940 902
583095022 787397816 1042454619015
306072127 154565592 325604613614
4645630 290029151 224269562739
614435986 478787009 684152300877
135901996 706598843 383429755608
954273199 309249448 820148773457
808231018 749281988 827416515626
980310985 522918612 1304797297473
94021187 921863196 769677785904
835260927 474708379 913290558789

7 85 871
2893191 739478943 590802191721
810774428 85962464 122895434706
680505400 806104933 656252756808
426611337 115659733 125200182625
820028483 952166315 512611763882
436909443 20773073 38287378321
129039332 53458145 39284077174

8 625 788
765290566 298853578 462534884643
747785007 174805727 434262735864
177540923 878960621 766752414044
881193450 375160964 635961514519
181432826 584657137 536545699594
331863363 801330842 767401441248
27468189 472196673 353291628460
916106615 800311933 852211589364

6 278 295
741174229 615967877 322298312758
963660894 734299472 427639146279
254255877 107320413 86948125734
390276401 474961261 204231593209
599686647 200154894 157533606510
329646495 502405263 215174292765

10 242 731
584168767 318931729 293898571984
934238461 510582413 578747179086
204577099 302378168 250446632917
714228442 589741224 504412350704
858562970 640883847 538539524523
342912875 984290074 782696607645
297564769 925306693 743275798542
159466862 966572716 685566436625
25170878 837805188 567260126898
293672546 218066301 206867110217

8 58 597
925661918 436358066 276717867602
675771528 940070589 526024806181
668021036 141765994 80683820726
682170415 51759828 67662229773
225058798 170681699 90521990706
762536947 665714221 415619011568
921299571 466082186 316634172688
252035155 433811990 257568096108

6 113 174
439515066 913907377 110611329666
177913330 828690118 92691985564
453029703 687278929 105165819396
184896141 678899601 59330866752
878395952 296071217 118929646365
583978819 298176325 83415235663

7 312 756
350515261 153302244 154942881995
768137543 852331569 701330360938
887669572 320445554 423508380410
748307461 458694125 411354868589
60169306 320754354 203525658358
391624447 374840316 381284429094
489519858 779686998 659762687378

8 963 338
565411321 77161141 367433184071
106996137 75752187 46885920054
590140651 122642582 402156824140
450216370 989441455 766473453745
142464766 954328558 282311322707
854197060 176173157 783910595502
331506994 114148438 144248405352
264819033 381739598 356204792096

10 393 548
146283962 977089156 427171437702
855323680 414009643 317029694407
410964149 938790055 597208639227
583368850 701567614 478370998627
485379831 640617129 348132369535
330497691 283638738 266912404058
161925756 60816846 84095148312
796292155 738833766 650737898433
637390035 715833178 467190918170
979311613 108308943 435499222321

9 543 12
712496431 247835533 375769396780
281374212 121747088 117449326892
946290999 137130908 289066544307
30064349 269766396 18383448882
982234003 811407304 400089661705
434022077 117385759 169614651933
76457209 919587168 44837531136
230010104 507196932 79882466691
781923452 25092544 331783537006

8 452 201
863992448 35424061 335216899652
196476934 941108706 261993172596
37454979 501540360 117287593726
39498155 568763271 132141958975
755295172 812336233 455128264423
548258342 106342515 236805840711
950031773 961951438 617921319880
545358212 644192005 363677458515

5 739 149
997053755 799024541 717008578354
157747481 280931976 109358340552
510525825 167395237 330819552282
968692827 356267667 593587149362
777503031 394124191 567073452602

5 715 216
24340637 533359971 111345881777
845765045 127667379 608655714809
468088402 401183467 115819843947
826380176 820263556 382026588767
216313571 205534830 101967113129

10 757 294
299736833 612415019 212252899394
190011314 761989553 251016323515
27249722 166301759 43615547416
666221074 590899001 513671967962
754644084 717811057 755594956030
635063976 253743427 551595887965
931113560 109174640 631236959277
284508126 938849134 447234222191
548814352 285584802 443410875546
456546186 163257175 275921449861

7 948 598
291705488 467420212 514104863855
86061674 769806414 482646343488
99098512 15216997 96173111091
501195374 263552454 591408178297
711383439 233324773 773389900011
435807127 435423302 466801924951
18651799 41837074 39866717202
