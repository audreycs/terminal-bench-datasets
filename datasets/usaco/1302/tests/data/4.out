296
754
196
336
130
40
185
149
82
221
39
572
930
313
726
321
504
105
721
621
74
181
914
411
534
357
137
489
457
753
242
303
577
215
673
539
174
422
262
105
170
54
791
86
340
471
441
331
406
851
525
332
731
52
370
559
808
878
350
721
498
339
523
507
761
713
712
305
585
960
315
954
719
346
619
539
40
459
90
245
356
999
85
324
788
409
391
114
176
125
118
289
788
359
240
75
240
655
408
475
